### PR TITLE
Sidebar

### PR DESCRIPTION
**Pull request by Gitongamiriam**

### CODE REVIEW

**Review by sahdasamier — COMMENTED** [review]
you shouldn't push your Pr with any comments

**Comment by Gitongamiriam** [comment]
Sure next time I will clean the code.

File: src/Profile/index.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        onChange={(e) => setUserName(e.target.value)}
+      />
+    </Box> */}
+        {/* <Typography sx ={{padding:'10px'}} variant="h4" component="div">Nabil Abadalla</Typography> */}
+        {/* <IconButton>
+          <Edit />
+        </IconButton> */}
+        {/* <Typography variant="h5" component="div">Title</Typography> */}
+        <Box sx={{ mt: 3 }}>
+          <Typography variant="h6" component="div">Name </Typography>
```
Comment (sahdasamier) [inline src/Profile/index.tsx:74]: try to do it like design without box borders , you can use gride for that or make borders none

**Review by sahdasamier — CHANGES_REQUESTED** [review]

**Comment by sahdasamier** [comment]
but actually, great job on your first task , now I can say, welcome back my developer  @Gitongamiriam

**Comment by Gitongamiriam** [comment]
I will put everything you've said into consideration.  Thank you so much.
It means a lot to ***@***.***

**Pull request merged**